include/detect/detect_re2c.h

```diff
@@ -8,12 +8,6 @@
 #define DETECT_RE2C_YYCURSOR(ctx) (ctx)->pos
 #define DETECT_RE2C_YYMARKER(ctx) (ctx)->marker
 #define DETECT_RE2C_YYCTXMARKER(ctx) (ctx)->ctxmarker
-#define DETECT_RE2C_YYFILL(ctx, pend, need, maxfill)                    \
-    do {                                                                \
-        int rv;                                                         \
-        if (!!(rv = detect_re2c_yyfill(ctx, pend, need, maxfill)))      \
-            return (-rv);                                               \
-    } while (0)
```

Comment:
This shouldn't be probably moved from the header file.

Reply:
Why?

Reply:
Because it is a common macro.

Reply:
In my changes it is not so

Reply:
My point is that the macro is common enough to just use it. You're probably doing something wrong if you have to do your special version of the macro.

Reply:
This is understandable and, unfortunately, a fact, but your other comment indicates this)